src/components/[guild]/Requirements/RoleRequirements.tsx

```diff
@@ -14,11 +14,18 @@ const VIRTUAL_LIST_REQUIREMENT_LIMIT = 10
 const PARENT_PADDING = "var(--chakra-space-5)"
 
 const RoleRequirements = ({ role }: Props) => {
-  const isVirtualList = role.requirements?.length > VIRTUAL_LIST_REQUIREMENT_LIMIT
-  const sliceIndex = (role.requirements?.length ?? 0) - 3
-  const shownRequirements = (role.requirements ?? []).slice(0, 3)
+  const requirements = role.hiddenRequirements
+    ? [
+        ...role.requirements,
+        { isHidden: true, type: "FREE" } as unknown as Requirement,
+      ]
+    : role.requirements
+
+  const isVirtualList = requirements?.length > VIRTUAL_LIST_REQUIREMENT_LIMIT
+  const sliceIndex = (requirements?.length ?? 0) - 3
+  const shownRequirements = (requirements ?? []).slice(0, 3)
   const hiddenRequirements =
```

Comment:
TODO for myself: I think I'll rename this variable, because the `hiddenRequirements` name might be confusing now that we have requirements which should be actually hidden (and not just hidden until the user doesn't click on the "and x more" button 😄